.github/workflows/backend.yml

```diff
@@ -62,15 +62,20 @@ jobs:
         run: |
           ./mvnw -T 2C -B checkstyle:check --no-snapshot-updates
       - name: Build and Package
-        run: |
-          ./mvnw -B install scalastyle:check \
-                 -Dmaven.test.skip=true \
-                 -Dcheckstyle.skip=true \
-                 -Dhttp.keepAlive=false \
-                 -Dmaven.wagon.http.pool=false \
-                 -Dmaven.wagon.http.retryHandler.count=3 \
-                 -Dmaven.wagon.httpconnectionManager.ttlSeconds=120 \
-                 --no-snapshot-updates
+        uses: nick-invision/retry@v2
+        with:
+          timeout_minutes: 10
```

Comment:
hi @asdf2014, `timeout_minutes: 10` may is not enough, seems most actions always beyond the limits.

Reply:
@leo65535 It is enough for this step